src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,90 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+// import { SearchLink } from '../SearchLink/SearchLink';
+import { TableHeader } from '../TableHeader';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleTable: FC<Props> = ({ people }) => {
+  const { personSlug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>
+            <TableHeader title="Name" />
+          </th>
```

Comment:
why don't you put <th> inside of component?

Reply:
I'm not sure that I understand correct. I use TableHeader component for every <th> that will work like button